build.rs

```diff
@@ -0,0 +1,56 @@
+use rerun_except::rerun_except;
+
+use std::env;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+static BOEHM_REPO: &str = "git://github.com/ivmai/bdwgc.git";
+static BOEHM_ATOMICS_REPO: &str = "git://github.com/ivmai/libatomic_ops.git";
+static BOEHM_DIR: &str = "bdwgc";
+static POINTER_MASK: &str = "-DPOINTER_MASK=0xFFFFFFFFFFFFFFF8";
```

Comment:
This should be guarded for 64-bits only I think.

Reply:
Good idea. I have added this here: d9eed7f12a3f5eeea4380ae5b108445a987739a8